index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Apple Website</title>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link href="https://fonts.googleapis.com/css?family=Lato:300,400&display=swap" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+	<nav class="top-nav">
+		<ul>
+			<li><i class="fa fa-apple" aria-hidden="true"></i></li>
+			<li>Store</li>
+			<li>Mac</li>
+			<li>iPod</li>
+			<li>iPhone</li>
+			<li>iPad</li>
+			<li>iTunes</li>
+			<li>Support</li>
+			<li class="searchbox-con"><i class="fa fa-search"></i><input type="search" name="search" class="searchbox"></li>
+		</ul>
+	</nav>
+
+	<main>
+		<section class="top-header">
+			<a href="#"><div class="first-image"></div></a>
+			<a href="#"><div class="second-image"></div></a>
+			<a href="#"><div class="third-image"></div></a>
+
+		</section>
+		
+	</main>
+
+	<footer>
+		<section class="promo-pages">
+			<div class="col first-promo-img"></div>
+			<div class="col second-promo-img"></div>
+			<div class="col third-promo-img"></div>
+			<div class="col fourth-promo-img"></div>
```

Comment:
Hi this is good every works fine althought these divs  here are suppose to be links that is wrapped within an a tag like I stated in the last review. You could correct that and submit this project. :+1:  goodluck on your next project :+1: